client/package.json

```diff
@@ -15,8 +16,9 @@
   },
   "scripts": {
     "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
-    "start-js": "react-scripts start",
+    "watch-css":
+      "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
+    "start-js": "HOST=battlesnake.local react-scripts start",
```

Comment:
why the env var here? this will probably require a /etc/hosts entry right?

Reply:
I can take this out - was left in on accident

Reply:
guessing it's for cookies